src/utils/react.js

```diff
@@ -12,3 +13,21 @@ export let wrapDisplayName = (name, Component) => Wrapped => {
   Wrapped.displayName = `${name}(${getDisplayName(Component)})`
   return Wrapped
 }
+
+// these are for futil :)
+// (x -> y) -> string -> {x} -> y
+let getWith = _.curry((customizer, path, object) =>
+  customizer(_.get(path, object))
+)
+// ({a} -> {b}) -> {a} -> {a, b}
+let expandObject = _.curry(transform => obj => ({ ...obj, ...transform(obj) }))
```

Comment:
For the curry to work, transform and obj should be on the same function

Reply:
🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀